lib/imap.php

```diff
@@ -87,31 +85,31 @@ public function checkPassword($uid, $password) {
 					$groups[] = $pieces[1];
 		}
 
-		$rcube = new imap_rcube();
-
-		$params = ["port"=>$this->port, "timeout"=>10];
+		$protocol = $this->sslmode ? "imaps" : "imap";
+		$url = "{$protocol}://{$this->mailbox}:{$this->port}";
+		$ch = curl_init();
+		curl_setopt($ch, CURLOPT_URL, $url);
+		curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
```

Comment:
do we need this option set to true?
If I get it right this returns the actual response in case of success instead of just `true` (for a failure it always returns `false`... is this needed?

Reply:
> … return value of curl_exec() instead of outputting it directly.
It is actually needed, without it, it shows the response directly and does not proceed with executing further.

Reply:
Thanks for clarifying!